Indico/Exception/NotFoundException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Indico.Exception
+{
+    [Serializable]
+    public class NotFoundException : System.Exception
```

Comment:
I think this should inherit from `RuntimeException` so the code is backward compatible.